packages/simcore-sdk/src/simcore_sdk/node_ports_common/r_clone.py

```diff
@@ -121,16 +110,4 @@ async def sync_local_to_s3(
             shlex.quote(f"{file_name}"),
         )
 
-        try:
-            await _async_command(*r_clone_command, cwd=f"{source_path.parent}")
-        except Exception as e:
-            logger.warning(
-                "There was an error while uploading %s. Removing metadata", s3_object
-            )
-            await delete_file(
-                session=session,
-                file_id=s3_object,
-                location_id=store_id,
-                user_id=user_id,
-            )
-            raise e
+        await _async_command(*r_clone_command, cwd=f"{source_path.parent}")
```

Comment:
wrong. the action is taken in the filemanager.py.
in the next PR it will be rightfully reverted to the last version in case a file was already there.